src/main/java/com/spring/demo/services/ItemServiceDiskImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.spring.demo.services;
+
+import com.spring.demo.models.Item;
+import com.spring.demo.repos.ItemDiskRepo;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ItemServiceDiskImpl implements ItemService {
+
+  private final ItemDiskRepo itemDiskRepo;
+
+  @Autowired
+  public ItemServiceDiskImpl(ItemDiskRepo itemDiskRepo) {
+    this.itemDiskRepo = itemDiskRepo;
+  }
+
+  @Override
+  public List<Item> getItems(String id) {
+    if (id != null && !id.isEmpty() && !id.isBlank()) {
```

Comment:
using util method to check empty input
`org.apache.commons.lang3.StringUtils.isEmpty`